src/platform/qt/MemorySearch.ui

```diff
@@ -312,23 +275,98 @@
        </attribute>
       </widget>
      </item>
-     <item row="3" column="1">
+     <item row="4" column="1">
       <widget class="QCheckBox" name="searchROM">
        <property name="text">
         <string>Search ROM</string>
        </property>
       </widget>
      </item>
+     <item row="3" column="1">
+      <widget class="QComboBox" name="encoding">
+       <property name="enabled">
+        <bool>false</bool>
+       </property>
+       <item>
+        <property name="text">
+         <string>ASCII</string>
+        </property>
+       </item>
+       <item>
+        <property name="text">
+         <string>UTF-8</string>
+        </property>
+       </item>
+       <item>
+        <property name="text">
+         <string>UTF-16LE</string>
+        </property>
+       </item>
+       <item>
+        <property name="text">
+         <string>Shift-JIS</string>
+        </property>
+       </item>
+      </widget>
+     </item>
     </layout>
    </item>
-   <item row="2" column="0" colspan="2">
-    <widget class="QDialogButtonBox" name="buttonBox">
-     <property name="standardButtons">
-      <set>QDialogButtonBox::Close</set>
+   <item row="0" column="1">
+    <layout class="QHBoxLayout" name="horizontalLayout">
+     <item>
+      <widget class="QLabel" name="msg">
+       <property name="text">
+        <string>MSG:</string>
+       </property>
+      </widget>
```

Comment:
This seems like a really bad design. Why do you want a status message? Just to show that the results are trimmed? Why trim the results then?